WebRoot/js/zimbraMail/briefcase/view/ZmPreviewPaneView.js

```diff
@@ -385,6 +403,19 @@ ZmPreviewPaneView.prototype._listSelectionListener = function(ev, item) {
     	} else {
     		this._previewView.set(item);
     	}
+    } else if(!this._controller.isReadingPaneOn() && item){
```

Comment:
I think we need to put folder condition as well. We need to allow only 'Files shared with me' folder's item to check the item is deleted or not.

Reply:
added the condition.